src/ctapipe_io_lst/__init__.py

```diff
@@ -437,40 +467,76 @@ def create_subarray(tel_id=1, reference_location=None):
 
     def fill_from_cta_r1(self, array_event, zfits_event):
         tel_id = self.tel_id
+        scale = self.data_stream.waveform_scale
+        offset = self.data_stream.waveform_offset
+        pixel_id_map = self.camera_config.pixel_id_map
+
+        # FIXME: missing modules / pixels
```

Comment:
So this will not work yet with  data with missing modules?

Reply:
No, not yet, see my comment above. 

I'd like to separate that part into a second PR, with proper dummy test data and so on. This one will get pretty large if I want to include everything.